tests/test_auto_machine.py

```diff
@@ -0,0 +1,71 @@
+# tests the auto feature of whecho using an environment variable $TEST_URL (please set yourself)
+
+import os
+import subprocess
+import re
+import socket
+import platform
+import toml
+
+# use different locations for different OSes
+CONFIG_PATH = {'Linux': os.path.expanduser('~/.config/.whecho/config.toml'),
```

Comment:
retrieve CONFIG_PATH from `--debug` flag.

this test will cause issues in the future if CONFIG_PATH location were to change in the future